pkg/cloudprovider/azure/object_store.go

```diff
@@ -18,121 +18,26 @@ package azure
 
```

Comment:
let's rename this file -- maybe `restic.go` or just `azure.go`

Reply:
Does it make sense to move this to a package closer to restic, e.g. `pkg/restic/azure`?

Reply:
yeah, that works for me

Reply:
Works for me too, glad you made this suggestion.